Composer/packages/client/src/recoilModel/selectors/project.ts

```diff
@@ -271,12 +274,9 @@ export const projectDialogsMapSelector = selector<{ [key: string]: DialogInfo[]
   },
 });
 
-export const projectTreeSelectorFamily = selectorFamily<
-  TreeDataPerProject[],
-  { showLgImports: boolean; showLuImports: boolean }
->({
+export const projectTreeSelectorFamily = selector<TreeDataPerProject[]>({
```

Comment:
You should rename this now that it's no longer a family (just `projectTreeSelector` is fine).